barretenberg/cpp/src/barretenberg/translator_vm/translator.test.cpp

```diff
@@ -16,8 +16,33 @@ using OpQueue = ECCOpQueue;
 auto& engine = numeric::get_debug_randomness();
 
 class TranslatorTests : public ::testing::Test {
+    using G1 = g1::affine_element;
+    using Fr = fr;
+    using Fq = fq;
+
   protected:
     static void SetUpTestSuite() { bb::srs::init_crs_factory(bb::srs::get_ignition_crs_path()); }
+
+    // Construct a test circuit based on some random operations
+    static CircuitBuilder generate_test_circuit(const Fq& batching_challenge_v,
```

Comment:
Here I'm just moving this sample circuit generation to a method so I can reuse it in the fixed VK test